src/components/Forms/EditPerson.jsx

```diff
@@ -0,0 +1,192 @@
+import React, { useState, useEffect } from 'react';
+import { withRouter, useHistory } from 'react-router-dom';
+import axios from 'axios';
+
+// App imports
+import Auth from 'Auth';
+import CreatePerson from 'CreatePerson';
+import { formatDate } from 'Utils/date';
+import { PEOPLE_URL } from 'Constants/ApiConstants';
+import { PEOPLE_PAGE_URL } from 'Constants/ClientConstants';
+
+
+const EditPerson = (props) => {
+  const history = useHistory();
+  const personId = props.location.state.peopleId;
+  const [personData, setPersonData] = useState();
+  const [formData, setFormData] = useState();
+  const [errors, setErrors] = useState({});
+
+  // reformat dates & peopleType into individual items for form field display
+  const reformatFields = (data) => {
+    const tempObj = data;
+    const splitDoB = data.dateOfBirth.split('-');
+    const splitDocumentExpiryDate = data.documentExpiryDate.split('-');
```

Comment:
you can use destructuring for some easy to read code:

```suggestion
    const [documentExpiryDateYear, documentExpiryDateMonth, documentExpiryDateDay] = data.documentExpiryDate.split('-');
```

Reply:
refactored reformatFields and reformatDate